lib/CMakeLists.txt

```diff
@@ -18,10 +18,15 @@ set(PROJECT_SOURCES
     src/values/abstract_config_value.cc
     src/values/config_boolean.cc
     src/values/config_number.cc
-    src/values/config_int.cc
```

Comment:
Changed my mind on further inspection. We now make the distinction between ints and longs, choosing ints whenever possible based on number size, including when a whole-number double is passed. This reflects what was done in the Java.